packages/fx-core/src/common/featureFlags.ts

```diff
@@ -81,3 +79,49 @@ export function isNewProjectTypeEnabled(): boolean {
 export function isOfficeJSONAddinEnabled(): boolean {
   return isFeatureFlagEnabled(FeatureFlagName.OfficeAddin, false);
 }
+
+///////////////////////////////////////////////////////////////////////////////
+// Notes for Office Addin Feature flags:
+// Case 1: TEAMSFX_OFFICE_ADDIN = false, TEAMSFX_OFFICE_XML_ADDIN = false
+//   1.1 project-type option: `outlook-addin-type`
+//   1.2 addin-host: not show but use `outlook` internally
+//   1.3 capabilities options: [`jason-taskpane`, `outlook-addin-import`]
+//   1.4 programming-language options: [`typescript`] (skip in UI)
+//   1.5 office-addin-framework-type: not show question but use `default_old` internally
+//   1.6 generator class: OfficeAddinGenerator
+//   1.7 template link: config.json.json-taskpane.default_old.typescript
+// Case 2: TEAMSFX_OFFICE_ADDIN = false AND TEAMSFX_OFFICE_XML_ADDIN = true
+//   2.1 project-type option: `office-xml-addin-type`
+//   2.2 addin-host options: [`outlook`, `word`, `excel`, `powerpoint`]
+//   2.3 capabilities options:
+//     if (addin-host == `outlook`) then [`json-taskpane`, `outlook-addin-import`]
+//     else if (addin-host == `word`) then [`word-taskpane`, `word-xxx`, ...]
+//     else if (addin-host == `excel`) then [`excel-taskpane`, `excel-xxx`, ...]
+//     else if (addin-host === `powerpoint`) then [`powerpoint-taskpane`, `powerpoint-xxx`, ...]
+//   2.4 programming-language options:
+//     if (addin-host == `outlook`) then [`typescript`] (skip in UI)
+//     else two options: [`typescript`, `javascript`]
+//   2.5 office-addin-framework-type options: not show but use `default` internally
```

Comment:
I think there comes the issue,

When `TEAMSFX_OFFICE_ADDIN = false AND TEAMSFX_OFFICE_XML_ADDIN = true` and is Outlook option, here value should be `default_old`, which should be same to the value in case 1.

After the test I carried, create project faild when I try create outlook taskpane from case 2.